Bigtable/src/Instance.php

```diff
@@ -0,0 +1,331 @@
+<?php
+/**
+ * Copyright 2018, Google LLC All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+namespace Google\Cloud\Bigtable;
+
+use Google\Cloud\Bigtable\Admin\V2\BigtableInstanceAdminClient as InstanceAdminClient;
+use Google\Cloud\Bigtable\Admin\V2\Instance_Type;
+use Google\Cloud\Bigtable\Admin\V2\Instance_State;
+use Google\Cloud\Bigtable\Admin\V2\StorageType;
+use Google\Cloud\Bigtable\Connection\ConnectionInterface;
+use Google\Cloud\Core\Exception\NotFoundException;
+use Google\Cloud\Core\Iterator\ItemIterator;
+use Google\Cloud\Core\LongRunning\LongRunningConnectionInterface;
+use Google\Cloud\Core\LongRunning\LongRunningOperation;
+use Google\Cloud\Core\LongRunning\LROTrait;
+
+/**
+ * Represents a Cloud Bigtable instance
+ *
+ * Example:
+ * ```
+ * use Google\Cloud\Bigtable\BigtableClient;
+ *
+ * $bigtable = new BigtableClient();
+ *
+ * $instance = $bigtable->instance('my-instance');
+ * ```
+ *
+ * @method LongRunningOperation resumeOperation() {
```

Comment:
This needs to be removed.

Reply:
The current process for getting an operation's name looks like this:

```php
$instance = $bigtable->instance('my-instance3');
$operation = $instance->create([
    $bigtable->buildClusterMetadata('my-cluster3', 'us-east1-b', Instance::STORAGE_TYPE_UNSPECIFIED, 1)
]);
doSomethingWith($operation->name());
```

I can definitely see the confusion on this looking like a quickstart guide since it is documented towards the top of the class. We actually have been using this in order to document public methods which are inherited from traits (in this case the [LROTrait](https://github.com/GoogleCloudPlatform/google-cloud-php/blob/master/Core/src/LongRunning/LROTrait.php)) - there is nothing special about it over the other methods.

Reply:
gotcha.  That's super confusing.